stratum/pipelines/ptf/BUILD

```diff
@@ -25,10 +25,10 @@ filegroup(
     srcs = ["scapy_exec.py"]
 )
 
-python_grpc_library(
-    name = "ptf_py_lib",
-    deps = ["@com_google_googleapis//google/rpc:code_proto"],
-)
+# python_grpc_library(
+#     name = "ptf_py_lib",
+#     deps = ["@com_google_googleapis//google/rpc:code_proto"],
+# )
```

Comment:
Before we generate our own python library for `code.proto`. but now we can use `code_py_proto` from upstream directly.